### PR TITLE
PBI report updates for interim schema

### PR DESCRIPTION
**Pull request by MSBrett**
## 🛠️ Description
Updated PBI reports utilizing the interim unified schema for MCA and EA channels.

Fixes #(issue)

## 🔬 How has this been tested?

- [x] 🫰 PS -WhatIf / az validate
- [x] 👍 Manually deployed + verified
- [ ] 💪 Unit tests

### CODE REVIEW

**Review by flanakin — CHANGES_REQUESTED** [review]

File: src/templates/finops-hub/reports/CommitmentDiscounts.pbix
[the remark this answers is not in the document]
Comment (MSBrett) [reply src/templates/finops-hub/reports/CommitmentDiscounts.pbix:1]: Done

File: src/templates/finops-hub/reports/CostSummary.Dataset/diagramLayout.json [hunk 1]
```diff
@@ -10,36 +10,36 @@
...
           },
           "zIndex": 0
         },
         {
           "location": {
-            "x": 0,
+            "x": 3175.51140422367,
             "y": 0
           },
-          "nodeIndex": "FOCUS_0.5",
```
[the remark this answers is not in the document]
Comment (MSBrett) [reply src/templates/finops-hub/reports/CostSummary.Dataset/diagramLayout.json:28]: Done

File: src/templates/finops-hub/reports/CostSummary.pbix
Comment (flanakin) [inline src/templates/finops-hub/reports/CostSummary.pbix:1]: Remove file

**Review by flanakin — CHANGES_REQUESTED** [review]

File: src/templates/finops-hub/reports/CommitmentDiscounts.Dataset/.pbi/unappliedChanges.json
Comment (flanakin) [inline src/templates/finops-hub/reports/CommitmentDiscounts.Dataset/.pbi/unappliedChanges.json:1]: I believe this is for changes that haven't been applied yet. Can you apply everything to make sure it all works?

File: src/templates/finops-hub/reports/CostSummary.Dataset/.pbi/unappliedChanges.json
Comment (flanakin) [inline src/templates/finops-hub/reports/CostSummary.Dataset/.pbi/unappliedChanges.json:1]: Apply changes

**Review by flanakin — CHANGES_REQUESTED** [review]

File: src/templates/finops-hub/reports/CommitmentDiscounts.Dataset/.pbi/unappliedChanges.json
Comment (MSBrett) [reply src/templates/finops-hub/reports/CommitmentDiscounts.Dataset/.pbi/unappliedChanges.json:1]: Done

File: src/templates/finops-hub/reports/CostSummary.Dataset/.pbi/unappliedChanges.json
Comment (MSBrett) [reply src/templates/finops-hub/reports/CostSummary.Dataset/.pbi/unappliedChanges.json:1]: Done

**Review by flanakin — APPROVED** [review]

**Pull request merged**